cmd/neofs-adm/docs/subnetwork.md

```diff
@@ -0,0 +1,116 @@
+
+Private key that has been passed with `--key` flag is the key of the owner of the just created subnetwork.
+
+You can check if your subnetwork was created successfully:
+
+```shell
+$ neofs-adm morph subnet get \
+    -r <side_chain_RPC_endpoint> \
+    -a <subnet_script_hash> \
+    --key </path/to/owner/private/key> \
+    --subnet <subnet_ID>
+Owner: NUc734PMJXiqa2J9jRtvskU3kCdyyuSN8Q
+```
+Your owner will differ from the example.
+
+## Step 2 (optional): Add node administrator
+
+Node administrators are the accounts that can manage (add and delete nodes) whitelist of the nodes that could be included
+to the subnetwork. Only subnet owner is allowed to add and remove node administrators from subnetwork.
+
+```shell
+$ neofs-adm morph subnet admin add \
+    -r <side_chain_RPC_endpoint> \
+    -a <subnet_script_hash> \
+    --key </path/to/owner/private/key> \
+    --admin <HEX_admin_public_key> \
+    --subnet <subnet_ID>
+Add admin request sent successfully.
+```
+
+## Step 3 (optional): Add node
+
+Adding a node to a subnetwork means that the node becomes able to service containers that have been created in that
+subnetwork. Addition only changes list of the allowed nodes. Node is not required to be bootstrapped at the moment of its
+inclusion.
+
+```shell
+$ neofs-adm morph subnet node add \
+    -r <side_chain_RPC_endpoint> \
+    -a <subnet_script_hash> \
+    --key </path/to/node_admin/private/key> \
+    --node <HEX_node_public_key> \
+    --subnet <subnet_ID>
+Add node request sent successfully.
+```
+
+**NOTE:** owner of the subnetwork is also allowed to add nodes.
+
+## Step 4 (optional): Add client administrator
+
+Client administrators are the accounts that can manage (add and delete nodes) whitelist of the clients that can creates
+containers in the subnetwork. Only subnet owner is allowed to add and remove client administrators from subnetwork.
+
+```shell
+$ neofs-adm morph subnet admin add \
+    -r <side_chain_RPC_endpoint> \
+    -a <subnet_script_hash> \
+    --key </path/to/owner/private/key> \
+    --admin <HEX_admin_public_key> \
+    --subnet <subnet_ID> \
+    --client \
+    --group <group_ID>
+Add admin request sent successfully.
+```
+
+**NOTE:** you do not need to create group explicitly, it would be created right after the first client admin has been
+added. Group ID is 4-byte positive integer number.
+
+## Step 4 (optional): Add client
+
+```shell
+$ neofs-adm morph subnet client add \
+    -r <side_chain_RPC_endpoint> \
+    -a <subnet_script_hash> \
+    --key </path/to/client_admin/private/key> \
+    --client <client_ownerID> \
+    --subnet <subnet_ID> \
+    --group <group_ID>
+Add client request sent successfully.
+```
+
+**NOTE:** owner of the subnetwork is also allowed to add clients. This is the only one command that accepts `ownerID`,
+not the public key. Administrator can manage only his group (a group where that administrator has been added by subnet
+owner).
```

Comment:
In the afterwords, I would love to see example how to configure storage node to enter subnet (couple of lines from yaml config). And one more example of neofs-cli command how to create container in the subnetwork. This is not directly connected to neofs-adm but it will cover all the cases to start using subnetworks.

Reply:
we discussed it a little and decided to make that file exactly "NeoFS subnetwork creation" instruction (e.g. it also does not contain examples of `deposit-notary` command but requires that). maybe it would be a good idea to have one small instruction (separate file) for one simple operation and allow them to have links to each other

Reply:
Then I invite you to add small instruction in separate file in this PR. `subnetwork-usage.md`, for example.

Reply:
reorganized structure a little: put all "managing" instructions to a separate file and also added `cli` and "bootstrapping" instructions to it